general/administration/hardware-acceleration.md

```diff
@@ -219,57 +242,46 @@ docker exec -it jellyfin ldconfig
 
 After that, you should ensure the NVIDIA driver loads correctly.
 
-### Configuring OpenCL Accelerated/VPP Tone Mapping
+### Configuring OpenCL / CUDA / Intel VPP Tone-Mapping
 
-OpenCL tone mapping with NVIDIA NVENC, AMD AMF, and Intel VA-API is through OpenCL image support.
+Hardware based tone-mapping with NVIDIA NVENC, AMD AMF, Intel QSV and VA-API is through OpenCL or CUDA.
 
-Full hardware based VPP tonemapping is supported on Intel VA-API and QSV on Linux.
+Intel hardware based VPP tone-mapping is supported on Intel QSV and VA-API on Linux. VPP is prefered when both two tone-mapping options are checked on Intel.
 
-OS/Platform | NVIDIA NVENC | AMD AMF  | Intel VA-API | AMD VA-API | Intel QSV | Software
+OS/Platform | NVIDIA NVENC | AMD AMF  | Intel QSV | Intel VA-API | AMD VA-API | Software
 ----------- | ------------ | -------- | ----------- | --------- | --------- | --------
-Linux       | OK           | OK       | OK          | planned   | OK        | planned
-Windows     | OK           | OK       | N/A         | N/A       | planned   | planned
-Docker      | OK           | untested | OK          | planned   | planned   | planned
+Linux       | ✔️          | ✔️       | ✔️        | ✔️          | ✔️   | WIP
+Windows     | ✔️           | ✔️       | ✔️        | N/A         | N/A       | WIP
+Docker      | ✔️           | untested | ✔️        | ✔️          | untested   | WIP
+
+> [!NOTE]
+> Tone-mapping on Windows with Intel QSV and AMD AMF requires Windows 10 or newer.
 
 > [!NOTE]
 > Make sure the hardware acceleration is well configured before reading this section.
 
-1. On Windows: Install the latest NVIDIA or AMD drivers.
+1. On Windows: Install the latest NVIDIA or AMD or Intel drivers.
 
 2. On Linux or docker:
-
-   - For NVIDIA cards, install `nvidia-opencl-icd` on Debian/Ubuntu:
-
-    ```sh
-    sudo apt update
-    sudo apt install nvidia-opencl-icd
-    ```
-
-    If presented with multiple package options, choose the one that matches the version of your current nvidia driver (`apt list --installed | grep nvidia`).
-
-    Install `opencl-nvidia` on Arch:
-
-    ```sh
-    sudo pacman -Sy opencl-nvidia
-    ```
-
    - For AMD cards, install `amdgpu-pro` with opencl arguments. See **Configuring AMD AMF encoding on Ubuntu 18.04 or 20.04 LTS** for more details.
 
     ```sh
     sudo ./amdgpu-pro-install -y --opencl=pal,legacy
+    sudo usermod -aG video $LOGNAME
+    sudo usermod -aG render $LOGNAME
```

Comment:
What should be substituted for `$LOGNAME`?

Reply:
This was copied from AMD's doc. LOGNAME => the name of the login user

Reply:
Should we calarify this or just keep it as is since the AMD docs need to be consulted anyway?

Reply:
Leave it as is. LOGNAME is a linux environment variable.
https://en.wikipedia.org/wiki/Logname